M2/Macaulay2/e/monideal.hpp

```diff
@@ -124,11 +124,7 @@ class MonomialIdeal : public EngineObject
   void text_out(buffer &o) const;
 
   const PolynomialRing *get_ring() const { return R; }
-  const Monoid *degree_monoid() const
-  {
-    const Ring *S = R;
-    return S->degree_monoid();
-  }
+  const Monoid *degree_monoid() const { return R->degree_monoid(); }
```

Comment:
Looking at this again, I'm not sure why `MonomialIdeal` needs a `degree_monoid()` method. If it's not used, can I remove it?